README.md

```diff
@@ -43,6 +49,25 @@ or
 npm install
 ```
 
+### Run Meilisearch
+
+You can run a Meilisearch instance locally with the following command:
+
+```
+docker run -it --rm \
+    -p 7700:7700 \
+    -e MEILI_MASTER_KEY='masterKey'\
+    -v $(pwd)/meili_data:/meili_data \
+    getmeili/meilisearch:v0.28.0 \
+    meilisearch --env="development"
+```
+
+### Import movies
+
+Run the following script in order to create the different indexes and upload the movies documents:
+
+`yarn setup`
+
 ## Environment variables
```

Comment:
I see you updated `.env.example` but this file is not going to be used when running in development. The user will still need to change the file to `.env`. Is this what we want? 
If not, we could change the `searchKey` here:

https://github.com/meilisearch/demo-movies/blob/4b35a36dc442cfeebc726207882115dbc2baaf13/src/pages/index.js#L18

with `masterkey` so that the user does not need to setup the env variables. Wdyt ?

Reply:
I thought it would be easier for contributors to update an `.env` file instead of digging into the code to find where the `master/search key` was set, since we don't know which `MEILI_MASTER_KEY` they will provide. WDYT ?

Reply:
If they use your docker command, their master key will be `masterKey`, so for me its seems logic that we are providing that as default key

Reply:
Resolved by removing the use of a master key for the setup ✨ 🦕